requests_cache/response.py

```diff
@@ -119,6 +121,24 @@ def revalidate(self, expire_after: ExpirationTime) -> bool:
         self.expires = self._get_expiration_datetime(expire_after)
         return self.is_expired
 
+    @property
+    def size(self) -> str:
+        """Get the size of the response body in human-readable format"""
+        n_bytes = len(self.content) if self.content else 0
+        return format_file_size(n_bytes)
```

Comment:
I still think you should move `format_file_size` to `__str__`.

it would be consistent with `format_datetime` being used there as well.

Reply:
Sure, that's been updated now.